libs/core/execution/include/hpx/execution/algorithms/as_sender.hpp

```diff
@@ -136,6 +136,27 @@ namespace hpx::execution::experimental {
 
             std::decay_t<Future> future_;
 
+#ifdef HPX_HAVE_STDEXEC
+            template <bool IsVoid, typename _result_type>
+            struct _set_value
```

Comment:
Let's not start with the leading underscore madness...

Reply:
right, will rename to `set_value_void_checked` or something similar. I've implemented something this in other parts of the code so I will make sure the names match. I do not think it is worth it to factor this out since it's only 2 or 3 places.